Makefile

```diff
@@ -9,7 +9,7 @@ lint:
 test:
 	./load/load_csvs.sh
 	dbt build
-	pytest tests
+	dbt test
```

Comment:
awesome!!! Could you also remove pytest dependencies in the Dockerfile + requirements.txt?